src/auto-evo/simulation/food_source/ChunkFoodSource.cs

```diff
@@ -38,11 +46,24 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
 
         score /= energyBalance.FinalBalanceStationary;
 
+        var ponderedEnergyGenerationScore = energyCompounds.Sum(
+            c => EnergyGenerationScore(microbeSpecies, c.Key) * ProportionOfTotalEnergy(c.Key));
```

Comment:
Is there a reason this needs to be proportional? I think you can save some division and seven code lines by just multiplying by amounts.

Remember that fitness scores are only used in ratio of other fitness scores of the same type, so scalar multipliers shouldn't matter.

Reply:
That was a point on which I wasn't sure, so I went the safe way. So we agree that I can add/multiply any constant to the score, that won't matter, right?